### ISSUE TITLE
Theme the buttonbar with CSS

### ISSUE THREAD

**Issue by tintou** [marcelklehr/socialcalc#34]
Right now the properties are duplicated and the hover/clicked effect are using Javascript when CSS is enough for this

**Comment by eddyparkinson** [comment]
@ramezrafla  did you convert the toolbar to CSS? 

>We may use SVG with symbols (take a look at trumbowyg) or just plain CSS.

from:
https://github.com/marcelklehr/socialcalc/issues/17

**Comment by ramezrafla** [comment]
Yes we have, but our fork has deviated quite a bit from this repo. We'll push to github shortly and you guys can copy paste the pieces you want.
